vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -56,21 +80,21 @@ the task list for that page.
  
 The default PAM implementation exposes the top level task fields in the task list for selection. It is possible to fetch task and process variables from the task list for rendering, but this is disabled by default to optimize performance.
  
-### Task Details
+#### Task Details
  
 The Task Details MFE renders detailed information about a given task in a read only grid. The task details widget is intended to give the end user processing a
```

Comment:
The task details widget is intended to give the end user, processing a task, the information necessary

Reply:
ok reading that i totally didn't understand it - like "end user," "processing a task," were part of a list being given something. if commas are technically necessary i'll have to rephrase!

Reply:
I think 'processing a task' is added detail, since the sentence still works without it. "the task details widget is intended to give the end user the information necessary..."

Reply:
changed to "The task details widget is intended to give the end user the information necessary to process the task."